packages/docs/components/Examples/mention/CustomComponentMentionEditor/CustomComponentMentionEditor.tsx

```diff
@@ -45,9 +45,12 @@ export default function CustomComponentMentionEditor(): ReactElement {
   const onOpenChange = useCallback((_open: boolean) => {
     setOpen(_open);
   }, []);
-  const onSearchChange = useCallback(({ value }: { value: string }) => {
```

Comment:
Maybe we should leave the original function without the `trigger` as in the non multi mention trigger use cases the option is irrelevant and for `defaultSuggestionsFilter` optional? This comment is for all samples which are not using multi mentions.

Reply:
Leave original function without trigger for non multi mention trigger